pkg/internalapis/cnsvolumeinfo/v1alpha1/cnsvolumeinfo_types.go

```diff
@@ -28,6 +28,12 @@ type CNSVolumeInfoSpec struct {
 
 	// vCenterServer is the IP/FQDN of the vCenter host on which the CNS volume is accessible.
 	VCenterServer string `json:"vCenterServer"`
+
+	// ID of the storage policy
+	StoragePolicyId string `json:"storagepolicyid,omitempty"`
+
+	// Name of the storage policy
```

Comment:
Isn't it better to call this `StorageClassName`?

Reply:
Policy ID would be better here. @shalini-b what benefit do you see for storing  stroageclassname?

Reply:
As per the discussion and design proposal, we agreed on adding storageClassName in the CnsVolumeInfo CR. This is need since, 2 SCs can be associated with StoragePolicy